src/components/organisms/LazyDrawer/LazyDrawer.tsx

```diff
@@ -23,6 +23,7 @@ const LazyDrawer: React.FC<IProps> = props => {
       visible={visible}
       width="400"
       onClose={onClose}
+      headerStyle={{backgroundColor: '#293235'}}
```

Comment:
Will ask @julioramosest about changing the panel header and body background colours. Because they look different in the design.